LanguageFeatures/Inline-classes/dynamic_semantics_member_invocation_A02_t01.dart

```diff
@@ -30,12 +31,10 @@ inline class IC<T> {
   final T id;
   IC(this.id);
 
-  Map<K, V> asMap<K, V extends T>(K key) => {key: this.id as V};
+  Map<K, Type> asMap<K, V extends T>(K key) => {key: V};
```

Comment:
Perhaps include `someOtherKey: T` here as well? The point is that the binding of `T` to a value is the really new thing here. We should test that the value of `K` and `V` are also as expected, but we certainly shouldn't ignore `T`.

(Also, when the description says `T1 .. Ts`, it means `T`, not `K, V`.)